src/utils/variables.scss

```diff
@@ -0,0 +1,16 @@
+$-color-background: rgb(112, 166, 224);
+$-color-shadow: rgba(232, 232, 240, 0.64);
+$-color-background-todo: rgb(228, 245, 246);
+$-color-white: white;
+$-color-error: red;
```

Comment:
Consider using a more descriptive name for the color red. `$-color-error` is not very descriptive. Consider using something like `$danger-color` or `$alert-color`. 🚨